elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -115,6 +126,9 @@ def prepare_tag(html_tags, index, group=None):
         elif html_tags.tag == 'pl-block-group':
             if grading_method != 'dag':
                 raise Exception('Block groups only supported in the "dag" grading mode.')
+            if partial_credit_type != 'none':
+                raise Exception('Partial credit not yet supported in questions using block groups')
```

Comment:
Nit for consistency:
```suggestion
                raise Exception('Partial credit not yet supported in questions using block groups.')
```

Reply:
sure